public/js/components/AtomEdit/CustomEditors/ExplainerEditor.js

```diff
@@ -0,0 +1,24 @@
+import React, {PropTypes} from 'react';
+import {ManagedForm, ManagedField} from '../../ManagedEditor';
+import FormFieldTextInput from '../../FormFields/FormFieldTextInput';
+import FormFieldsScribeEditor from '../../FormFields/FormFieldScribeEditor';
+
+export class ExplainerEditor extends React.Component {
+
+  static propTypes = {
+    atom: PropTypes.shape({type: PropTypes.string, id: PropTypes.string}).isRequired,
```

Comment:
is there a reason for not using `atomPropType` here?

Reply:
Ah, no. Let me look at adding it in.